CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## XX.XX.XX - 2023-XX-XX
+* [Changed][7999](https://github.com/stripe/stripe-android/pull/7999) In test mode, PaymentSheet fails to load when getting saved payment methods fails.
```

Comment:
```suggestion
* [Changed][7999](https://github.com/stripe/stripe-android/pull/7999) In test mode, PaymentSheet now fails to load when getting saved payment methods fails.
```

Tweak just to clarify that this is _new_ behavior. As written, could be misunderstood as referring to a bug that we fixed or something